build/common.props

```diff
@@ -0,0 +1,23 @@
+<Project>
+  <Import Project="..\version.props" />
```

Comment:
This needs `$(MSBuildThisFileDirectory)`. Includes are always relative to the "entry" project file.